kano_qt_apps/lib/src/app_list.cpp

```diff
@@ -59,7 +58,7 @@ void AppList::add_app(JSON_Object *new_app, std::shared_ptr<App> fallback)
 
 void AppList::add_app_from_file(std::string file_path, std::shared_ptr<App> fallback)
 {
-    this->add_app(App(file_path), std::move(fallback));
+    this->add_app(App(file_path, std::move(fallback)));
```

Comment:
Did you mean to move this `)`? It looks reasonable but I wonder how this ever worked...